server/controllers/stock/reports/stock/consumption_graph.js

```diff
@@ -0,0 +1,87 @@
+const {
+  _, db, util, ReportManager, pdfOptions, STOCK_CONSUMPTION_GRAPTH_TEMPLATE,
+} = require('../common');
+
+const FilterParser = require('../../../../lib/filter');
+/**
+   * @method stockEntryReport
+   *
+   * @description
+   * This method builds the stock entry report as either a JSON, PDF, or HTML
+   * file to be sent to the client.
+   *
+   * GET /reports/stock/consumption_graph
+   */
+function stockConsumptionGrathReport(req, res, next) {
+  let report;
+
+  const params = _.clone(req.query);
+
+  const optionReport = _.extend(params, pdfOptions, {
+    filename : 'REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE',
+    waitFor : 1500,
+  });
+
+  // set up the report with report manager
+  try {
+    report = new ReportManager(STOCK_CONSUMPTION_GRAPTH_TEMPLATE, req.session, optionReport);
+  } catch (e) {
+    return next(e);
+  }
+
+  const sql = `
```

Comment:
Would it be possible to just use the stock functions that already exist in our stock modules for finding lots and consumption?  I don't want there to be a different in logic between this SQL and those SQLs.  Plus, this doubles the chance of bugs.